cg/constants/constants.py

```diff
@@ -246,6 +246,12 @@ class MicrosaltAppTags(StrEnum):
     PREP_CATEGORY: str = "mic"
 
 
+class MutantQC:
```

Comment:
Minor naming, if you think it makes sense consider renaming the enum to `MutantQCThresholds` and removing `THRESHOLD` from the enum members.

Reply:
I think this enum naming make the functions that use them more readable. For example:

```shell
def is_valid_total_reads_for_external_negative_control(reads: int) -> bool:
    return reads < MutantQC.EXTERNAL_NEGATIVE_CONTROL_READS_THRESHOLD
```

Here reads should be below the threshold.